docs/blocks-storage/querier.md

```diff
@@ -277,6 +277,18 @@ blocks_storage:
     # CLI flag: -blocks-storage.s3.bucket-lookup-type
     [bucket_lookup_type: <string> | default = "auto"]
 
+    # How many times to retry a failed AWS S3 operation.
+    # CLI flag: -blocks-storage.s3.operation-retries
+    [operation_retries: <int> | default = 1]
+
+    # The minimum backoff time when retrying failed AWS S3 operation.
+    # CLI flag: -blocks-storage.s3.retry-min-backoff
+    [retry_min_backoff: <duration> | default = 10s]
+
+    # The maximum backoff time when retrying failed AWS S3 operation.
+    # CLI flag: -blocks-storage.s3.retry-max-backoff
+    [retry_max_backoff: <duration> | default = 1m]
+
```

Comment:
Does this needs to be configurable? 

We are trying to have fewer configs so if we could avoid adding configs that probably nobody will change i think its better...

@yeya24 WDYT?

Reply:
Agree. Let's find a sane default value and expose these configs in the future if needed.